README.md

```diff
@@ -100,11 +100,6 @@ To start a masternode:
 $ mn start
 ```
 
-To start a full node:
```

Comment:
I guess it still worth documenting this. We could add a separate section for that after ”Register mssternode”

Reply:
Added a bit further below, since this is not technically a command